app/models/subscriber_list.rb

```diff
@@ -51,4 +55,9 @@ def link_values_are_valid
   def gov_delivery_config
     EmailAlertAPI.config.gov_delivery
   end
+
+  # We could make this more sophisticated if needed, e.g. check document_type.
+  def use_email_alert_frontend_for_email_collection?
+    ENV["USE_EMAIL_ALERT_FRONTEND_FOR_EMAIL_COLLECTION"] == "true"
```

Comment:
I wonder if we just want to check this env var exists? or is there prior art we can apply about boolean env vars

I have a memory but couldn't find anything very quickly that when we have a boolean env var it set in hieradata it comes out as a 1. This may feel slightly weird if we have to set it in hieradata intentionally wrapped as a string.

I could be wrong though :-/

Rest looks good 👍

Reply:
I agree, I think we should just check for the existence of this environment variable. It means we can leave it as `undef` in Puppet until we want it set.

Reply:
Ok, sounds good to me. Updated. 👍